protocol/0056-REWA-rewards_overview.md

```diff
@@ -51,7 +53,82 @@ These metrics apply only to the sum of fees for the epoch in question.
+Fee-based reward metrics (the total fees paid/received by each party as defined above) are stored in [LNL checkpoints](./0073-LIMN-limited_network_life.md) and are restored after a checkpoint restart to ensure rewards are not lost.
+
+### Average position metric
+
+The average position metric, $m_{ap}$, measures each parties time-weighted average position over a number of epochs.
+
+At the start of each epoch, the network must reset each parties time weighted average position for the epoch ($\bar{P}$) to `0`. Whenever a parties position changes during an epoch, **and** at the end of the epoch, this value should be updated as follows.
+
+Let:
+
+- $\bar{P}$ be the parties time weighted average position in the epoch so far
+- $P_{n}$ be the parties position before their position changed
+- $t_{n}$ be the time the party held the previous position in seconds
+- $t$ be the amount of time elapsed in the current epoch so far
+
+
+$$\bar{P} = \bar{P} \cdot \left(1 - \frac{t_{n}}{t}\right) + \frac{|P_{n}| \cdot t_{n}}{t}$$
+
+At the end of the epoch, the network must store the parties time weighted average position and then calculate their average position reward metric as follows.
+
+Let:
+
+- $m_{ap}$ be the parties average position reward metric
+- $\bar{P_{i}}$ be the parties time weighted average position in the i th epoch
+- $N$ be the window length specified in the recurring transfer.
+
+$$m_{ap} = \frac{\sum_{i}^{n}\bar{P_{i}}}{N}$$
+
+### Relative return metric
+
+The relative return metric, $m_{rr}$, measures each parties average relative return, weighted by their [time-weighted average position](#average-position-metric), over a number of epochs.
+
+At the end of each epoch, the network must calculate and store the parties relative returns as follows.
+
+Let:
+
+- $r_i$ be the parties relative returns in the epoch
+- $m2m_{wins}$ be the sum of all mark-to-market win transfers in the epoch
+- $m2m_{losses}$ be the sum of all mark-to-market loss transfers in the epoch
+- $TWAP$ be the parties time-weighted average position in the epoch.
+
+$$r = \frac{m2m_{wins} + m2m_{losses}}{TWAP}$$
+
+And calculate their average relative returns over the last $N$ epochs as follows.
+
+Let:
+
+- $m_{rr}$ be the parties relative return reward metric
+- $r_i$ be the parties change in pnl in the i th epoch
+- $N$ be the window length specified in the recurring transfer.
+
+$$m_{rr} = \max(\frac{\sum_{i}^{n}{r_{i}}}{N}, 0)$$
+
+### Returns volatility metric
+
+The return volatility metric, $m_{rv}$, measures the volatility of a parties returns across a number of epochs.
+
+At the end of an epoch, if a party has had net returns less than or equal to `0` over the last $N$ epochs (where $N$ is the window length specified in the recurring transfer), their reward metric $m_{rv}$ is set to `0`. Otherwise, the network should calculate the standard deviation of the set of each parties returns over the last $N$ epochs.
```

Comment:
We should connect negative net returns to the above calc of $r_i$ to make sure there is no ambiguity.